settings.gradle.kts

```diff
@@ -54,3 +54,4 @@ include(":tools:noopfactory")
 include(":tools:javabackport")
 include(":tools:lint")
 include(":tools:benchmark")
+include(":tools:benchmarksample")
```

Comment:
> [!NOTE]
> I'm gonna be that guy again 😅, Out of consistency, I'd make it a module `:sample:benchmark` instead

Reply:
Actually I am thinking about the same thing, now updated.